spec/resource/collection_spec.rb

```diff
@@ -230,19 +230,19 @@
 
         expect(directory.groups.limit(30).count).to eq(100)
 
-        expect(directory.groups.limit(30).current_page.items.count).to eq(30)
+        expect(directory.groups.limit(30).size).to eq(30)
 
         expect(directory.groups.limit(30).offset(30).count).to eq(70)
 
-        expect(directory.groups.limit(30).offset(30).current_page.items.count).to eq(30)
+        expect(directory.groups.limit(30).offset(30).size).to eq(30)
 
         expect(directory.groups.limit(30).offset(60).count).to eq(40)
 
-        expect(directory.groups.limit(30).offset(60).current_page.items.count).to eq(30)
+        expect(directory.groups.limit(30).offset(60).size).to eq(30)
 
         expect(directory.groups.limit(30).offset(90).count).to eq(10)
 
-        expect(directory.groups.limit(30).offset(90).current_page.items.count).to eq(10)
+        expect(directory.groups.limit(30).offset(90).size).to eq(10)
```

Comment:
sry, in a hurry to quickly make a commit to be reviewed i didn't update that part. yes the size is called on current_page which returns CollectionPage instance.